### PR TITLE
long running crash recovery test

### PR DESCRIPTION
**Pull request by zjma**
## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: consensus/src/rand/rand_gen/rand_store.rs [hunk 1]
```diff
@@ -15,7 +15,7 @@ use aptos_types::randomness::{RandMetadata, Randomness};
 use itertools::Either;
 use std::collections::{BTreeMap, HashMap, HashSet};
 
-const FUTURE_ROUNDS_TO_ACCEPT: u64 = 200;
+const FUTURE_ROUNDS_TO_ACCEPT: u64 = 20;
```
Comment (danielxiangzl) [inline consensus/src/rand/rand_gen/rand_store.rs:18]: Is 20 too aggressive?

Comment (zjma) [reply consensus/src/rand/rand_gen/rand_store.rs:18]: this is an aggressive chaos test to find something to break, so...

File: testsuite/smoke-test/src/randomness/long_running_crash_recovery.rs [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+"#, consensus_config_bytes, target_randomness_status);
+    let txn_result = aptos_cli.run_script(root_idx, script.as_str()).await.unwrap();
+    assert!(txn_result.success.unwrap());
+}
+
+async fn make_change(validator_power_status_vec: &mut Vec<bool>, rest_cli: &Client, aptos_cli: &CliTestFramework, root_idx: usize, swarm: &mut LocalSwarm) {
+    let stopped_nodes: Vec<usize> = validator_power_status_vec.iter().enumerate().filter_map(|(idx, &power_on)|if !power_on {Some(idx)} else {None}).collect();
+    match stopped_nodes.len() {
+        0 => {
+            match thread_rng().gen_range(0.0, 1.0) {
```
Comment (danielxiangzl) [inline testsuite/smoke-test/src/randomness/long_running_crash_recovery.rs:141]: Why float not integer for random number?

File: consensus/src/rand/rand_gen/rand_store.rs (hunk 1 above)
Comment (danielxiangzl) [reply consensus/src/rand/rand_gen/rand_store.rs:18]: But production will also use this config?

Comment (zjma) [reply consensus/src/rand/rand_gen/rand_store.rs:18]: i don't know. this change and this PR is temporary so far.

File: testsuite/smoke-test/src/randomness/long_running_crash_recovery.rs (hunk 2 above)
Comment (zjma) [reply testsuite/smoke-test/src/randomness/long_running_crash_recovery.rs:141]: not too much difference?

File: consensus/src/rand/rand_gen/rand_store.rs (hunk 1 above)
Comment (danielxiangzl) [reply consensus/src/rand/rand_gen/rand_store.rs:18]: It will I think

**Pull request closed without merging**